src/main/java/uk/gov/hmcts/sptribs/common/ccd/HighLevelDataSetupApp.java

```diff
@@ -32,6 +32,9 @@ public class HighLevelDataSetupApp extends DataLoaderToDefinitionStore {
         new CcdRoleConfig("caseworker-st_cic-senior-judge", "PUBLIC"),
         new CcdRoleConfig("caseworker-st_cic-judge", "PUBLIC"),
         new CcdRoleConfig("caseworker-st_cic-respondent", "PUBLIC"),
+        new CcdRoleConfig("hmcts-admin", "PUBLIC"),
```

Comment:
should we need hmcts-admin here too?

Reply:
just remembering, It might be needed for supplementary data. So, we might need this, just cross check.

Reply:
Not required for supplementary data.